app/controllers/bus_stops_controller.rb

```diff
@@ -11,7 +11,7 @@ def autocomplete
   def by_route
     @route = Route.find_by number: params.require(:number)
     if @route.present?
-      @stops = @route.bus_stops.order(:name)
+      @stops = @route.bus_stops.order(:completed).group_by(&:completed?)
```

Comment:
Just to refrain from writing `.group_by{|stop| stop.completed? == false}`. Honestly don't know which one is better.

Reply:
Why do you want the boolean to be backwards?

Reply:
So incomplete stops appear first

Reply:
Because incomplete stops have a higher priority for users than completed stops.